### DISCUSSION TITLE
Startup Screen Poll

### DISCUSSION THREAD

**Discussion by nate902011** [JordanSantiagoYT/FNF-JS-Engine#692] [Polls]
Out of all startup screens...
![sillyLogoQuestionning](https://github.com/user-attachments/assets/785bd6cf-d24f-478a-95fb-4ab2517db129)

**Comment by JordanSantiagoYT** [comment]
the 4 unadded startup screens:

![PolLogo](https://github.com/user-attachments/assets/985d472e-fe02-4efe-9e44-8545cab9fc71)
![lobotomy3](https://github.com/user-attachments/assets/47003476-0971-4797-812f-a44f2a938e35)
![sillyLogoOld](https://github.com/user-attachments/assets/3cc2317c-595c-4d84-891c-e77c3109e73b)
![lobotomy2](https://github.com/user-attachments/assets/2b681fc8-d8b9-4772-9f1c-cba3dd0b4c2a)
